mypy.ini

```diff
@@ -0,0 +1,13 @@
+[mypy]
+  python_version = 2.7
```

Comment:
I'm actually thinking about droppping Python 2 support. The package is simple enough that the current version (without type annotations) would work fine with Python 2 for the foreseeable future. (And after that Python 2 is unsupported anyways.)

Or maybe lets wait a while and release this version so that Python 2 users still have an annotated version to use. And in the next major release afterwards we can switch to real Python 3 type annotations.